docs/source/awesome-trio-libraries.rst

```diff
@@ -0,0 +1,57 @@
+Awesome Trio Libraries
+======================
+
+.. List of Trio Libraries
+
+   A list of libraries that support Trio, similar to the awesome-python
+   list here: https://github.com/vinta/awesome-python/
+
+
+.. currentmodule:: trio
+
+You have completed the turorial, and are enthusiastic about building
+great new applications and libraries with async functionality.
+However, to get much useful work done you will want to use some of
+the great libraries that support trio-flavoured concurrency. This list
+is not complete, but gives a starting point. Another great way to find
+trio-compatible libraries is to search on PyPI for the `Framework :: trio`
```

Comment:
The single-backticks here are causing the CI build to fail: https://travis-ci.org/python-trio/trio/jobs/600224047

I think you want double-backticks, because ReST is annoying like that.

Also, let's capitalize Trio.